plenum/server/primary_elector.py

```diff
@@ -357,7 +357,7 @@ def processPrimary(self, prim: Primary, sender: str) -> None:
                     logger.debug("{} selected primary on the basis of {}".
```

Comment:
Its a lower level detail describing on what basis election results were calculated so does not indicate status of anything, why do you think we should have it on the CLI, the logs prior to this should be enough to indicate the result of election (`selected primary ...`)